frontend/model/contracts/group.js

```diff
@@ -647,6 +715,15 @@ sbp('chelonia/defineContract', {
           // handles proposal pass or fail, will update proposal.status accordingly
           proposals[proposal.data.proposalType][result](state, message)
           Vue.set(proposal, 'dateClosed', meta.createdDate)
+
+          // TODO: update 'streaks.noVotes'
```

Comment:
Is this different from `groupStreaks.noVotes` below?

Reply:
@taoeffect No the below logic is what this comment was meant for. forgot to delete it after updating it.